quadlets/docker/application.properties

```diff
@@ -0,0 +1,33 @@
+# Database Config
+spring.datasource.url=jdbc:mariadb://10.88.0.2:3306/cfe_18
```

Comment:
Similar to what @kortemik posted in https://github.com/teragrep/cfe_18/pull/97#discussion_r1930544443

Where is this sourced from?